test/functional/developer_portal/buyer/stats_controller_test.rb

```diff
@@ -53,7 +53,7 @@ def setup
         get :index
 
         doc = Nokogiri::HTML.parse(response.body)
-        assert_equal [@live_app1, @live_app2], @controller.instance_variable_get('@cinstances')
+        assert_equal [@live_app1, @live_app2], @controller.instance_variable_get('@applications')
```

Comment:
You can use `assigns(:applications)` instead ... BUT functional tests are deprecated

https://github.com/rails/rails/issues/18950 https://github.com/rails/rails/blob/35e0190ac54ec852f8ff5b9196ccf8a24e8d193b/actionpack/lib/action_controller/test_case.rb#L29

Reply:
I know, I am not adding any test, just removing the unused variable and changing the super old test for the variable that we are actually using 😄

Reply:
Use `assigns` at least to get rid of the ugly `instance_variable_get`